internal/flink/command_shell.go

```diff
@@ -23,7 +24,8 @@ func (c *command) newShellCommand(cfg *config.Config, prerunner pcmd.PreRunner)
 	}
 
 	c.addComputePoolFlag(cmd)
-	cmd.Flags().String("identity-pool", "", "Identity pool ID.")
+	cmd.Flags().String("identity-pool", "", "Identity pool ID (deprecated).")
```

Comment:
This is still EA, so can we just make a breaking change here? It would be odd if we removed the feature flag and this flag was marked as deprecated.

Reply:
This is a breaking change and identity pools will not be supported in OP. This is due to change of direction from a few months ago regarding how we do auth. We have to mark this as deprecated and remove it soon

Reply:
> We have to mark this as deprecated and remove it soon

:+1: As long as it's removed before GA!

Reply:
Can you create and link a ticket here for removing this flag before GA (if you don't plan to remove it in this PR), so we don't forget?

Reply:
Added a todo and linked to the ticket. https://confluentinc.atlassian.net/browse/KFS-941